ethereum/eip712/eip712_test.go

```diff
@@ -453,17 +453,17 @@ func (suite *EIP712TestSuite) verifyPayloadMapAgainstFlattenedMap(original map[s
 	suite.Require().True(ok)
 
 	messages, ok := interfaceMessages.([]interface{})
-	suite.Require().True(ok)
+	if ok {
```

Comment:
Can we please add a comment here explaining why a !ok is accepted?

Reply:
done